packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1652,7 +1652,6 @@ idx.error.code.access_denied.device_assurance.remediation.chrome.chromeos.key_tr
 # End user remediation messages for Android Zero Trust signals
 idx.error.code.access_denied.device_assurance.remediation.android.zero.trust.play_protect_required = Please ensure Google Play Protect is enabled and that a scan has been performed, then try accessing this application again.
 idx.error.code.access_denied.device_assurance.remediation.android.zero.trust.play_protect_scan_threshold = There is a problem with your Google Play Protect system scan. Please open Google Play and take action on the warnings displayed.
-idx.error.code.access_denied.device_assurance.remediation.android.zero.trust.device_integrity_not_met = Device integrity not met.
```

Comment:
We determined this message is not needed, removed from backend as well. This was only utilized by AZT which is currently in Product_Dev status and no Prod orgs have this enabled.